backend/tarsy/agents/iteration_controllers/react_tools_controller.py

```
@@ -0,0 +1,171 @@
+                prompt = self.prompt_builder.build_data_collection_react_prompt(prompt_context, react_history)
+                
+                # REUSE centralized ReAct system message
+                messages = [
+                    LLMMessage(
+                        role="system", 
+                        content=self.prompt_builder.get_standard_react_system_message("comprehensive data collection for this stage")
+                    ),
+                    LLMMessage(role="user", content=prompt)
+                ]
+                
+                response = await self.llm_client.generate_response(messages, context.session_id, context.agent.get_current_stage_execution_id())
```

Ruff (0.12.2)

Line too long (144 > 88)

(E501)